server/src/main/java/com/objectcomputing/checkins/services/memberprofile/currentuser/CurrentUserServicesImpl.java

```diff
@@ -24,4 +30,15 @@ public MemberProfile findOrSaveUser(@NotNull String name, @NotNull String workEm
                 "", workEmail, "", null, "")));
 
     }
+
+    public MemberProfile currentUserDetails() {
```

Comment:
When doing authentication it is probably easier to include a Principal argument in the controller than to use the security service.
https://micronaut-projects.github.io/micronaut-security/latest/guide/#retrievingAuthenticatedUser